documentation/modules/rn-2.5.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-2.5_{context}"]
+= {project-full} 2.5
+
+You can use {the-lc} {project-first} to migrate virtual machines from the following source providers to {virt} destination providers:
+
+* VMware vSphere
+* {rhv-full} ({rhv-short})
+* {osp}
+* Open Virtual Appliances (OVAs) that were created by VMware vSphere
+* Remote {virt} clusters
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-25_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Migration using OpenStack
+
+In this version, migration using OpenStack source providers graduated from a Technology Preview feature to a fully supported feature.
+
+.Disabling FIPS
+
+EMS enforcement is disabled for migrations with VMware vSphere source providers in order to enable migrations from versions of vSphere that are supported by {project-full} but do not comply with the 2023 FIPS requirements.
+
+.Integration of the create/update provider dialog
+
+Reworked the forms of create and update providers so that they will align with the typical look and feel within the OCP console and ensure the up-to-date data is displayed.
+
+.Standalone UI
+
+The old UI of MTV 2.3 cannot be enabled by setting `feature_ui: true` in ForkliftController anymore.
+
+[id="new-features-and-enhancements-25_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.Migration using OVA files that are compatible with VMware vSphere
+
+// i need to wait for this ticket to be merged to add a link
+In {project-short} {project-version}, you can migrate using Open Virtual Application (OVA) files that were created by VMware vSphere as source providers. It may also work with OVAs that are created by other systems and compatible with vSphere.
+
+.Migrating VMs between OCP clusters
+
+// i need to wait for this ticket to be merged to add a link
+// apologies, I meant link to output of the doc ticket rather than https://issues.redhat.com/browse/MTV-336[Import VMware-compatible OVAs]
+// But let me add this as a placeholder 
+You can import Virtual Machine (VMs) without having access from {ocp-name} to the source system the VMs were running on. The VMs are exported to a place that is accessible to {ocp-name}, optionally accessible to the source system at the same time.
+
+.Migration of VMs with direct LUNs from RHV 
+
+During the migration from RHV, direct LUNs are detached from the source virtual machines and attached to the target virtual machines. Note that this mechanism does not work yet for Fibre Channel. link:https://issues.redhat.com/browse/MTV-329[MTV-329]
+
+.Additional authentication methods for OpenStack
```

Comment:
The additional authentication methods are not working properly now, should be add the bug https://issues.redhat.com/browse/MTV-701 and tell user the workaround?

Reply:
actually, maybe it better be documented as a known issue and add a link to the CLI section there

Reply:
I have added a workaround. We can move in to the known issues next week

Reply:
ack